distributor.xsd

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified"
+           targetNamespace="https://phar.io/composer-distributor" xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:rep="https://phar.io/composer-distributor">
+
+  <xs:element name="distributor" type="rep:distributorType"/>
+
+  <xs:complexType name="distributorType">
+    <xs:sequence minOccurs="1" maxOccurs="unbounded">
+      <xs:element type="rep:pharType" name="phar"/>
+    </xs:sequence>
+    <xs:attribute type="xs:string"  name="packageName" use="required"/>
+    <xs:attribute type="xs:string"  name="keyDirectory"/>
+  </xs:complexType>
+
+  <xs:element name="phar" type="rep:pharType"/>
+
+  <xs:complexType name="pharType">
+    <xs:attribute type="xs:string" name="name" use="required"/>
+    <xs:attribute type="xs:string"  name="file" use="required"/>
```

Comment:
We cant't because of the %PLACEHOLDER% stuff :(
We would have to define a custom type.

Reply:
Is that due to the `%`? If that is the issue I'd happily use something else as placeholder-marker. Like `::` or `#` or `$` or `@`

Reply:
Not sure because %PLACEHOLDER% is quite common, the next common thing would be {$PLACEHOLDER} I think (that would work in `anyURI`)

Reply:
What are we discussing here? Why should there be a %PLACEHOLDER%?

Reply:
URI can have placeholders in them so we can download the PHAR
- http://github.com/releases/ %VERSION% /tool.phar
- http://github.com/releases/2.16.4/tool.phar

If we with the xsd definition from `xsd:string` to `xsd:anyURI` `%placeholder%` is not allowed because of the `%`
So we where thinking about changing the placeholder syntax or defining a custom type in the xsd.

Reply:
And we have to replace the version number at runtime, right?

Reply:
We don't have to, since you have to tag the mediating package as well we could force people to update the config file.

Reply:
But I would argue that allowing placeholders makes it much easier because you do not have to update the `distributor.xml` for every release. With placeholders you can just put the new `tag` on the same `commit`.

Reply:
That was the initial idea. To be able to have as little work as possible with the plugin. Re-tagging should be enough.

Reply:
Wouldn't that be `${PLACEHOLDER}`

Reply:
I changed the placeholder syntax to `{$PLACEHOLDER}` and set the xsd type to `xs:anyURI`

Reply:
Ahhh I have no preference, I could change it to `${PLACEHOLDER}` in no time.

Reply:
Probably depends on what syntax / language style we want to follow.

In a PHP string, it would be `$varname`, whith `{$varnbame}` being an alternative.
In BASH, ANT's `build.xml` and alike it's `${varname}`.

Reply:
Since it's a PHP lib I would argue to go with `{$varname}` but... like I said... no preference here :)

Reply:
`{{varname}}`

/me runs

Reply:
Fine for me as well :P 
5 4 3 2 ...

Reply:
I somehow don't like the `$` in the URL ;)
So, unless @heiglandreas totally disagrees, let's go for mustache-style `{{varname}}`

Reply:
1 done placeholder syntax is now `{{vorname}}`